src/api/dataSource/NodeDataSource.ts

```diff
@@ -105,7 +105,7 @@ export class NodeDataSource {
     this.client = client;
   }
 
-  async getInstalledApplications(): Promise<Application[]> {
+  async getInstalledApplications(): ApiResponse<ListApplicationsResponse> {
```

Comment:
I don't think we should wrap a Promise inside ApiResponse:
```
Promise<ApiResponse<Application[]>>

Reply:
cc @MatejVukosav

Reply:
@xilosada Check this part of code https://github.com/calimero-network/admin-dashboard/blob/master/src/pages/Applications.tsx#L85